File: app/src/main/java/com/jheank16oz/questionform/place/AutocompleteResult.java
package com.jheank16oz.questionform.place;

import java.util.Iterator;
import java.util.List;

/**
 * Created by ICATECH on 8/05/18.
 * Developer Jhean Carlos Piñeros Diaz
 */

public class AutocompleteResult implements Iterable<Prediction> {

    public List<Prediction> predictions;

    @Override
    public Iterator<Prediction> iterator( ) {
        return this.predictions.iterator( );
    }

    public int size( ) {
        return this.predictions.size( );
    }

}

File: questionform/src/main/java/com/jheank16oz/questionform/place/AutocompleteResult.kt
package com.jheank16oz.questionform.place

/**
 * Created by ICATECH on 8/05/18.
 * Developer Jhean Carlos Piñeros Diaz
 */

class AutocompleteResult : Iterable<Prediction> {

    var predictions: List<Prediction>? = null

    override fun iterator(): Iterator<Prediction> {
        return this.predictions!!.iterator()
    }

    fun size(): Int {
        return this.predictions!!.size
    }

}
